delphi-frontend/src/main/java/au/com/integradev/delphi/symbol/resolve/NameResolver.java

```diff
@@ -664,9 +664,9 @@ private static boolean isDynamicArrayReference(Type type) {
     return type.isClassReference() && ((ClassReferenceType) type).classType().isDynamicArray();
   }
 
-  private void readPossibleUnitNameReference(NameReferenceNode node) {
+  private void readPossibleUnitNameReference(NameReferenceNode node, boolean inAttribute) {
```

Comment:
Let's call it `attribute` to be consistent with the parameter name in `readNameReference`.

Reply:
I actually regret naming that parameter `attribute`, I feel like it's an unexpected name for a boolean to have. I've changed it instead to be consistently `inAttribute`, what do you think of that?

Reply:
Fine by me.